packages/clients/dish/src/utils/autocomplete.ts

```diff
@@ -80,10 +84,10 @@ export const autocomplete: DishAutocompleteFunction = (_, __, inputValue) => {
 /**
  * selecting autocomplete offer re-triggers search
  */
-export const selectResult: SelectResultFunction = (
-  { commit, dispatch },
-  { feature }
-) => {
+export const selectResult: SelectResultFunction<
+  AddressSearchState,
+  AddressSearchGetters
+> = ({ commit, dispatch }, { feature }) => {
```

Comment:
Please expose them correctly as mandated by this [ADR](https://github.com/Dataport/polar/wiki/Architecture-Decision-Records-%F0%9F%93%83#2024-03-05-how-to-expose-additional-exports-of-a-package)

Reply:
https://github.com/Dataport/polar/pull/163/commits/8a2ad1cf3fab8efd93e5201fd310bf59cc1c085c